third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -78,6 +78,20 @@ var (
 		"replica_configuration.0.username",
 		"replica_configuration.0.verify_server_certificate",
 	}
+<% unless version == 'ga' -%>
+	externalReplicaConflictKeys = []string{
+		"settings",
+		"replica_configuration",
+		"ip_address",
+		"server_ca_cert",
+		"connection_name",
```

Comment:
since `connection_name` is computed, it can't be set by users anyway, so it doesn't really need to be here (likewise for a few of the others)

Reply:
removed